### ISSUE TITLE
AMReX_Particles_F.H

### ISSUE THREAD

**Issue by jrood-nrel** [AMReX-Codes/amrex#596]
At some point `AMReX_Particles_F.H` went away. It looks like some particle functionality that was in PeleC was using:
```
Source/PeleC_advance.cpp:399: undefined reference to `SprayParticleContainer::insertParticles(double, int, int)
Source/PeleCParticles.cpp:278: undefined reference to `SprayParticleContainer::InitParticlesUniform(int const&, int const&)
```
Do these two functions still exist in some form, or are there equivalents to them that exist?

**Comment by atmyers** [comment]
Sorry for the breakage. `AMReX_Particles_F.H` contained the following functions:

```
    void amrex_particle_set_position (amrex_particle_real*, int ns, int np,
                                      AMREX_D_DECL(const amrex_real* x,
                                             const amrex_real* y,
                                             const amrex_real* z));

    void amrex_particle_get_position (const amrex_particle_real*, int ns, int np,
                                      AMREX_D_DECL(amrex_real* x,
                                             amrex_real* y,
                                             amrex_real* z));

    void amrex_build_nl(void*, int ns, int np,
                        int* nbor_list, int* size_nl, const int* lo, const int* hi,
                        const amrex_real* plo, const amrex_real* dx);

    void amrex_deposit_cic(const amrex_particle_real*, int ns, int np, int nc,
                           amrex_real* rho, const int* lo, const int* hi,
                           const amrex_real* plo, const amrex_real* dx);

    void amrex_deposit_particle_dx_cic(const amrex_particle_real*, int ns, int np, int nc,
                                       amrex_real* rho, const int* lo, const int* hi,
                                       const amrex_real* plo, const amrex_real* dx,
                                       const amrex_real* particle_dx);

    void amrex_interpolate_cic(const amrex_particle_real*, int ns, int np,
                               const amrex_real* acc, const int* lo, const int* hi, int ncomp,
                               const amrex_real* plo, const amrex_real* dx);

    void amrex_atomic_accumulate_fab(const amrex_real*, const int*, const int*,
                                     amrex_real*, const int*, const int*, int);
```

Of these, the first three were removed, while the others were converted to C++ and moved to different places. The deposit / interpolate routines now live in `AMReX_Particle_mod_K.H`, while `amrex_atomic_accumulate_fab` has been deprecated in favor of the equivalent method of `BaseFab` called `atomicAdd`. 

I'm not sure which of these routines are used by the Spray particles in Pele. The two routines you mention don't look they ever were defined inside amrex.

**Comment by jrood-nrel** [comment]
@atmyers Thanks for the detailed response. It appears I had a knee-jerk reaction when I commented out our `#include<AMReX_Particles_F.H>` and saw the missing symbols. I should have originally searched for these symbols instead of making an assumption they were in AMReX. They are in fact in a file in another one of our own repositories and I believe I have resolved this issue with no fault of AMReX. Sorry about any time spent on this!
